package.json

```diff
@@ -11,86 +11,87 @@
     "test": "tests"
   },
   "scripts": {
-    "build": "ember build",
-    "lint": "./node_modules/.bin/eslint ./*.js app config lib server tests",
+    "build": "ember build --environment=production",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*",
```

Comment:
Hey @allthesignals I was just looking this PR over one more time before merging and noticed this. Can you explain why you needed this change? I see that the `lint-staged` section in package.json is running `eslint --cache --fix` so this probably isn't relevant to that but just running `yarn run lint` locally spit out a bunch of errors.

Does this change allow things through the linter that weren't allowed through before? I'm not necessarily opposed to that for this PR if we need to, just want to understand what it's doing.

Reply:
Oh I just realized that this command is only linting the handlebars templates. The [package](https://www.npmjs.com/package/ember-template-lint) we use for that doesn't seem to have a `fix` option but I'm not as concerned about those files anyways. Thanks for the context.